pubsub/README.rst

```diff
@@ -19,174 +19,27 @@ Usage
 Subscriber
 ~~~~~~~~~~
 
-The official Google publisher returns a future which is mostly useable as-is.
-``gcloud-rest`` returns the future without change while ``gcloud-aio`` wraps it in ``asyncio`` terms.
+At the moment, ``gcloud-{aio,rest}-pubsub`` only provides a set of handles
+to call Pubsub REST API.
 
-An HTTP-oriented version, in keeping with the other ``gcloud-{aio,rest}-*`` libraries,
-will likely be coming soon -- though our current approach works reasonably well
-for allowing the official ``grpc`` client to be used under ``asyncio``, we
-continue to see threading oddities now and again which we've not been able to
-solve. As such, we do not wholeheartedly recommend using the
-``SubscriberClient`` of this library in production, though a resilient enough
-environment for your use-case may be possible.
-
-Here's the rough usage pattern for subscribing:
+Here's the rough usage pattern for using ``SubscriberClient``:
 
 .. code-block:: python
 
     from gcloud.aio.pubsub import SubscriberClient
     from gcloud.aio.pubsub import SubscriberMessage
 
     client = SubscriberClient()
-    # create subscription if it doesn't already exist
+    # create subscription
     client.create_subscription(
```

Comment:
Please `await` these in the docs (the `awaits` will get removed for the `gcloud-rest-*` version of the docs)